internal/provider/private_endpoint_services_resource_test.go

```diff
@@ -68,18 +68,18 @@ func TestIntegrationPrivateEndpointServicesResource(t *testing.T) {
 			client.Cluster{
 				Name:          clusterName,
 				Id:            uuid.Nil.String(),
-				CloudProvider: "AWS",
+				CloudProvider: "GCP",
 				Config: client.ClusterConfig{
 					Dedicated: &client.DedicatedHardwareConfig{
 						StorageGib:     15,
-						MachineType:    "m5.large",
+						MachineType:    "not verified",
```

Comment:
I'm not sure about this value. You could set it to `n2-standard-2`. Maybe leaving it unset also works.

Reply:
I updated it because I noticed that all the values used for GCP in terraform are using aws names.  We can go through and start validating them at some point if we want.